sovrin_client/test/cli/helper.py

```diff
@@ -160,6 +163,23 @@ def x(cli):
     return x
 
 
+def addTrusteeTxnsToGenesis(trusteeList, trusteeData, domainTransactionsFilePath):
+    added = 0
+    if trusteeList and len(trusteeList) and trusteeData:
+        with open(domainTransactionsFilePath, 'a') as domainFile:
+            index = 5
```

Comment:
It's for that first number in a transaction, not line number https://github.com/evernym/plenum/blob/master/data/transactions_local#L1